tests/download_test_data.py

```diff
@@ -17,6 +17,7 @@ def _provide_testing_data(dataset=None):
         'somato': ('https://github.com/sappelhoff/MNE-somato-data'),
         'ds000246': 'https://github.com/OpenNeuroDatasets/ds000246',
         'ds000248': 'https://github.com/OpenNeuroDatasets/ds000248',
+        'ds000248_autobads': 'https://github.com/OpenNeuroDatasets/ds000248',
```

Comment:
I would not create a new dataset. It will blow up the testing time if we start doing this

Reply:
Ok, will try to solve it by re-using the `ds000248` dataset. Creating a new dataset was just the quickest solution I could come up with. But I totally see your point.

Reply:
Resolved in 92b0090